src/python/WMCore/WMSpec/StdSpecs/PromptReco.py

```diff
@@ -61,7 +61,7 @@ def buildWorkload(self):
                                                splitAlgo = self.procJobSplitAlgo,
                                                splitArgs = self.procJobSplitArgs,
                                                stepType = cmsswStepType,
-                                               forceUnmerged = True)
+                                               forceUnmerged = ["write_ALCARECO"] if 'ALCARECO' in self.writeTiers else False)
```

Comment:
hmm, boolean forceUnmerged is used even in TaskChain.
With this fix you're basically redefining this argument, which will be source of confusion. On the other hand, having yet another argument for that list is also not that great...